packages/react-native-sdk/src/components/participants/VideoRenderer.tsx

```diff
@@ -0,0 +1,209 @@
+import React, { useEffect, useRef } from 'react';
+import { StyleSheet } from 'react-native';
+import { RTCView } from 'react-native-webrtc';
+import { ParticipantVideoType, ParticipantViewProps } from '.';
+import {
+  CallingState,
+  SfuModels,
+  StreamVideoParticipant,
+  VisibilityState,
+} from '@stream-io/video-client';
+import { useCall, useCallCallingState } from '@stream-io/video-react-bindings';
+import { useMediaStreamManagement } from '../../providers';
+import { Z_INDEX } from '../../constants';
+import { ParticipantVideoFallback as DefaultParticipantVideoFallback } from './ParticipantVideoFallback';
+
+/**
+ * Props to be passed for the VideoRenderer component.
+ */
+export type VideoRendererProps = Pick<
+  ParticipantViewProps,
+  'ParticipantVideoFallback'
+> & {
+  /**
+   * The video kind that will be displayed.
+   * @types `screen` or `video`
+   */
+  videoMode: ParticipantVideoType;
+  /**
+   * The participant whose info will be displayed.
+   */
+  participant: StreamVideoParticipant;
+  /**
+   * When set to false, the video stream will not be displayed even if it is available.
+   *
+   * @default false
+   */
+  muteVideo?: boolean;
+};
+
+/**
+ * Lower level component, that represents only the video part (wrapper around the WebRTC)
+ */
+export const VideoRenderer = ({
```

Comment:
We need to move videoFallback inside as a prop to video renderer

as per [Android](https://staging.getstream.io/video/docs/android/ui-cookbook/video-fallback/)

Reply:
videoFallback prop 

lives in `ParticipantView` then passed to `VideoRenderer`

Reply:
also in this way onLayout within this component